test/system/structure.test.js

```diff
@@ -96,7 +96,7 @@ const optionIds = [
     stackLimit: {
       name: 'Schema resolution nesting limit',
       type: 'integer',
-      default: 8,
+      default: 10,
```

Comment:
@umeshp7 There were other places where we also require this change. Take a look at this PR for more idea: https://github.com/postmanlabs/openapi-to-postman/pull/652

Reply:
Options.md - done
deref.js - done
options.js - done
schemUtils - done
structure.test.js - done
validator.test.js  - pending

Reply:
@VShingala 
```
{ suggestAvailableFixes: true, showMissingInSchemaErrors: true, optimizeConversion: false, stackLimit: 10 })
```
This^ change needs to be reverted right?

Reply:
Yes, it can be reverted. I had to do it as we were checking a part of body that only got resolved at 10. so if we're having 10 now we can revert it.